BudgetTracker.BudgetSquirrel.WebApi/Models/Request/Credentials.cs

```diff
@@ -0,0 +1,8 @@
+namespace BudgetTracker.BudgetSquirrel.WebApi.Models.Requests
```

Comment:
I would rename this to "Messages.Requests" rather than "Models.Request". Models, to me, are more of domain objects that describe an entity in the application. This is simply a couple fields that are structured... in other words, it's a message format.

Reply:
Yeah I am still working on this code. not fully done just wanted to get it out there. This is one of the things I refactored.